docs/source/features.rst

```diff
@@ -36,8 +36,19 @@ be https://www.usnews.com/opinion and https://www-stag.usnews.com/opinion,
 respectively. It tries to intelligently remove extra ".", "-", and "/" as
 needed if the level would be omitted in the case of ``live``.
 
-Note that the default level is ``live``, so running smoketest without a level
-argument is equivalent to running it with ``--level=live``.
+If your preproduction sites use the form https://stage.usnews.com or
+https://local.usnews.com instead of https://www-stag.usnews.com and similar,
+you can do:
+
+.. code-block:: yaml
```

Comment:
Is this supposed to go in `settings.yaml`? If so I think this section should mention that.

Reply:
Good point. Addressed in fb08106f3e6274aaa1fa8dc762c831bd1ecda196.